src/pseudopeople/interface.py

```diff
@@ -108,19 +105,19 @@ def _generate_dataset(
 
 def validate_source_compatibility(source: Path):
     # TODO [MIC-4546]: Clean this up w/ metadata and update test_interface.py tests to be generic
-    metadata = source / "metadata.yaml"
-    if metadata.exists():
-        version = _get_data_version(metadata)
-        if version > parse(DATA_VERSION):
+    changelog = source / "CHANGELOG.rst"
+    if changelog.exists():
+        version = _get_data_changelog_version(changelog)
+        if version > parse("1.4.2"):
             raise DataSourceError(
                 f"The provided simulated population data is incompatible with this version of pseudopeople ({psp_version}).\n"
                 "A newer version of simulated population data has been provided.\n"
                 "Please upgrade the pseudopeople package."
             )
-        if version < parse(DATA_VERSION):
+        if version < parse("1.4.2"):
             raise DataSourceError(
                 f"The provided simulated population data is incompatible with this version of pseudopeople ({psp_version}).\n"
-                "Current data version is {version} but a newer version {DATA_VERSION} is available.\n"
+                "The simulated population data has been corrupted.\n"
```

Comment:
Is "corrupted" the correct way to describe this case?

Reply:
This is how the comment was before. Corrupted in this sense is the user's data source has had something happen to it where it no longer works with their version of the package. Corrupted meaning unknown issue (from our end) that has happened on the user's end.